source/reference/minio-mc/mc-version-enable.rst

```diff
@@ -113,7 +113,7 @@ Behavior
 Bucket Versioning with Existing Data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Enabling bucket versioning on a bucket with existing data immediately applies a versioning ID to any unversioned object.
+Enabling bucket versioning on a bucket with existing data immediately creates a null value version ID for each unversioned object.
```

Comment:
```suggestion
Enabling bucket versioning on a bucket with existing data immediately creates a ``NULL`` value version ID for each unversioned object.
```

I think it's literally `NULL` as the value, to the point that you can operate on the object using that.